src/mixins/input.js

```diff
@@ -107,7 +107,7 @@ export default {
 
       if (this.id) data.attrs = { for: this.id }
 
-      return this.$createElement('label', data, this.label)
+      return this.$createElement('label', data, this.$slots.label || this.label)
```

Comment:
`this.label` is the props provided by user, which is used for simple string input, while label slot `this.$slots.label` is used for rendering more complex labels.